docs/src/components/mdx/headings.tsx

```diff
@@ -9,7 +9,7 @@ export const H1: React.FC<HTMLAttributes<HTMLHeadingElement>> = props => {
   const className = cx(styles.h1, props.className)
 
   return (
-    <Text.Heading size={Text.Heading.sizes.xLarge} className={className}>
+    <Text.Heading size={Text.Heading.sizes.xlarge} className={className}>
```

Comment:
This change should be reverted looks like xLarge casing didn't change on Heading.Sizes

Reply:
Fixed